docs/developer/payments.mdx

```diff
@@ -1131,6 +1131,172 @@ In case of a failure result, the [errors](api-reference/payments/objects/payment
+
+The `id` is the ID of the payment app received from [checkout.availablePaymentGateways](api-reference/checkout/objects/checkout.mdx#code-style-fontweight-normal-checkoutbavailablepaymentgatewaysbcodepaymentgateway--) or `shop.availablePaymentGateways`. The JSON `data` is passed to the payment app via the synchronous webhook.
+
+```graphql
+mutation PaymentGatewayInitializeTokenization(
+  $id: String!
+  $channel: String!
+  $data: JSON
+) {
+  paymentGatewayInitializeTokenization(
+    id: $id
+    channel: $channel
+    data: $data
+  ) {
+    result
+    data
+    errors {
+      field
+      code
+      message
+    }
+  }
+}
+```
+
+In the response, Saleor returns:
+
+- `result`: The enum value of [PaymentGatewayInitializeTokenizationResult](api-reference/payments/enums/payment-gateway-initialize-tokenization-result.mdx).
+- `data`: The JSON `data` returned by the payment app.
+- `id`: The ID of the payment method.
+
+More details about the triggered webhook [PAYMENT_GATEWAY_INITIALIZE_TOKENIZATION_SESSION](api-reference/webhooks/enums/webhook-event-type-sync-enum.mdx#code-style-fontweight-normal-webhookeventtypesyncenumbpayment_gateway_initialize_tokenization_sessionbcode) can be found [here](developer/extending/webhooks/synchronous-events/stored-payment-method.mdx#initialize-payment-gateway-session).
+
+#### Initializing payment method tokenization
+
+The [paymentMethodInitializeTokenization](api-reference/payments/mutations/payment-method-initialize-tokenization.mdx) mutation starts the process of tokenizing the payment method. This mutation triggers the synchronous webhook [PAYMENT_METHOD_INITIALIZE_TOKENIZATION_SESSION](api-reference/webhooks/enums/webhook-event-type-sync-enum.mdx#code-style-fontweight-normal-webhookeventtypesyncenumbpayment_method_initialize_tokenization_sessionbcode), which is sent to the app with the provided [id](api-reference/payments/mutations/payment-method-initialize-tokenization.mdx#code-style-fontweight-normal-paymentgatewayinitializetokenizationbidbcodestring--).
+
+The `id` is the ID of the payment app received from [checkout.availablePaymentGateways](api-reference/checkout/objects/checkout.mdx#code-style-fontweight-normal-checkoutbavailablepaymentgatewaysbcodepaymentgateway--) or [shop.availablePaymentGateways](api-reference/miscellaneous/objects/shop.mdx#code-style-fontweight-normal-shopbavailablepaymentgatewaysbcodepaymentgateway--). The JSON [data](api-reference/payments/mutations/payment-method-initialize-tokenization.mdx#code-style-fontweight-normal-paymentmethodinitializetokenizationbdatabcodejson-) is passed to the payment app via the synchronous webhook.
```

Comment:
In this paragraph, I would add a note what's `paymentFlowToSupport` supposed to do - i.e. if you want to use this payment method for 1-click checkout use `INTERACTIVE`, for charging customers later by staff use `NON_INTERACTIVE`

Reply:
Added